src/main/asciidoc/reference/elasticsearch-repository-queries.adoc

```diff
@@ -0,0 +1,157 @@
+[[elasticsearch.query-methods]]
+= Query methods
+
+[[elasticsearch.query-methods.finders]]
+== Query lookup strategies
+
+The Elasticsearch module supports all basic query building feature as String,Abstract,Criteria or have it being derived from the method name.
```

Comment:
There's an issue with spacing: String,Abstract,Criteria

Not sure if it's me, I do not understand that `String,Abstract,Criteria` part.

Reply:
that was already in this form in the docs, I changed it.